truss/contexts/image_builder/image_builder.py

```diff
@@ -80,5 +81,17 @@ def prepare_image_build_dir(self, build_dir: Path = None):
             with docker_file_path.open('w') as docker_file:
                 docker_file.write(dockerfile_contents)
 
+        readme_template_path = TEMPLATES_DIR / README_TEMPLATE_NAME
+
+        with readme_template_path.open() as readme_template_file:
+            readme_template = Template(readme_template_file.read())
+            # examples.yaml may not exist
+            # if examples.yaml does exist, but it's empty, examples_raw is None
+            examples_raw = self._spec.examples if self._spec.examples_path.exists() else None
+            readme_contents = readme_template.render(config=self._spec.config, examples=examples_raw)
+            readme_file_path = build_dir / MODEL_README_NAME
+            with readme_file_path.open('w') as readme_file:
+                readme_file.write(readme_contents)
```

Comment:
I think it's worth extracting the readme generation logic out into a separate function. We may want to expose the readme generation as truss command as well, but that can come later.